tasks/Module 4/task_2.py

```diff
@@ -0,0 +1,31 @@
+"""Fill in the Gaps."""
+import os
+
+
+def fill_gaps(folder_path, prefix):
+    """Fill in the Gaps."""
+    # init file list with first not needed element
+    file_list = [None]
```

Comment:
Why do you need `None` here?

Reply:
It is not necessarily should be None. I just dont need first element in this list (with index 0). 
If i dont do it then i will need to adjust shift between indices (starts from 0) and filenames (starts from 1). 
Or i can assume, that filenames also starts from 0. But it is not defined in task.

Reply:
When you apply solution from another comment (with `enumerate`) you will be fine without adjusting shift manually.

Reply:
you are so right. Thank you for this hint.